src/wstool/multiproject_cli.py

```diff
@@ -37,6 +37,7 @@
 import shutil
 import datetime
 import yaml
+import argparse
```

Comment:
I also prefer argparse to optparse, but we should not mix them, I think. Is there a specific reason you could not use optparse?